### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by istreeter**

### CODE REVIEW

File: config/config.hocon.sample [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    # Optional, set the  polling mode for retrieving records. Default is FanOut
+    # "retrievalMode": "FanOut"
+    # "retrievalMode": {
+    #   "type": "Polling"
+    #   "maxRecords": 1000
+    # }
+
+    # Optional override to disable cloudwatch metrics
+    # "disableCloudWatch": true
```
Comment (istreeter) [inline config/config.hocon.sample:27]: I wonder if this should be in a metrics section like for the [new s3 loader](https://github.com/snowplow/snowplow-s3-loader/blob/1165b7edd57bf6c207c2c60d9ca99a7bc793eaf7/config/config.hocon.sample#L70)

File: modules/loader/src/main/resources/application.conf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+snowplow {
```
Comment (chuwy) [inline modules/loader/src/main/resources/application.conf:1]: Can we use the same format as in main config? Or other way:

```
snowplow.input.retrievalMode = "FanOut"
snowplow.input.disableCloudWatch = false
```

Just the format of:

```
root {
  key = value
}
```

Feels very unusual.

File: config/config.hocon.sample (hunk 1 above)
Comment (chuwy) [reply config/config.hocon.sample:27]: I think yes and should be named the same. It's exactly the same setting.

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+Remove requirement to always use Kinesis Enhanced Fan Out (#14)
+Bump postgresql to 42.2.0 (#15)
+Bump schema-ddl to 0.13.0 (#27)
+Bump log4s to 1.10.0 (#26)
+Bump fs2 to 2.5.6 (#25)
+Bump doobie to 0.13.4 (#24)
+Bump fs2-google-pubsub to 0.17.0 (#23)
+Bump fs2 aws to 3.1.1 (#22)
+Bump circe to 0.14.1 (#30)
+Bump cats-effect to 2.5.0 (#29)
```
Comment (chuwy) [inline CHANGELOG:14]: 2.5.1 is available.

**Review by chuwy — APPROVED** [review]
Looks great!

I looked mostly at #12 and #31 and everything looks sensible there.

File: config/config.hocon.sample (hunk 1 above)
Comment (istreeter) [reply config/config.hocon.sample:27]: I actually copied `disableCloudWatch` from [stream enrich config](https://github.com/snowplow/enrich/blob/a47072f352d9b70f08fbc31b4d5fd78dbbf1e811/config/config.hocon.sample#L78) and [the elasticsearch loader config](https://github.com/snowplow/snowplow-elasticsearch-loader/blob/3ed0746c56f9be880259edc9971b87712d5830e7/examples/config.hocon.sample#L92) and [the old s3 loader config](https://github.com/snowplow/snowplow-s3-loader/blob/4157b9c768ec58d773c905cdd8d898ebccbf7c2c/examples/config.hocon.sample#L67)

Two of those three links will soon be obsolete anyway.  And the elasticsearch loader needs a redesign.  So I'm happy to say that was the old way of doing things, and going forwards we copy the new s3 loader.

File: config/config.hocon.sample [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
+  # Human-readable storage target name, used only for logging
+  "name": "Acme Ltd. Snowplow Postgres"
+  # Machine-readable unique identifier
+  "id": "5c5e4353-4eeb-43da-98f8-2de6dc7fa947"
+
+  "input": {
+    # Enable the Kinesis source
+    "type": "Kinesis"
```
Comment (benjben) [inline config/config.hocon.sample:9]: It would be nice to also show an example config for PubSub

**Review by benjben — APPROVED** [review]
Looks great !

File: .github/workflows/test.yml [hunk 5]
```diff
@@ -1,6 +1,13 @@
...
+name: CI
 
-on: [push, pull_request]
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - master
+      - develop
```
[the remark this answers is not in the document]
Comment (istreeter) [reply .github/workflows/test.yml:9]: I just copied [what we do in enrich](https://github.com/snowplow/enrich/blob/a47072f352d9b70f08fbc31b4d5fd78dbbf1e811/.github/workflows/ci.yml#L7)

I'm not strongly opinionated on this.  I often push work to development branches before they are ready to be tested, so there is no reason to waste github action minutes on those branches.  And if I ever do want to force the github action to run on a development branch, then I create a dummy commit that edits the workflow to enable that branch.

File: .github/workflows/test.yml [hunk 6]
```diff
@@ -1,6 +1,13 @@
...
+  pull_request:
```
[the remark this answers is not in the document]
Comment (istreeter) [reply .github/workflows/test.yml:10]: No it's deliberate, although I agree the syntax is weird.  It means run the action on any pull request.

**Pull request merged**